build.sbt

```diff
@@ -713,20 +713,62 @@ lazy val examples = (projectMatrix in file("examples"))
 
 val compileDocs: TaskKey[Unit] = taskKey[Unit]("Compiles docs module throwing away its output")
 compileDocs := {
-  (docs.jvm(scala2_12) / mdoc).toTask(" --out target/sttp-docs").value
+  (docs.jvm(scala2_13) / mdoc).toTask(" --out target/sttp-docs").value
 }
 
+val jeagerClientVersion = "1.0.0"
+val braveOpentracingVersion = "0.34.2"
+val zipkinSenderOkHttpVersion = "2.15.0"
```

Comment:
we have a whole section with versions in the build file, maybe we could put it there?

Reply:
I put them here since these aren't our dependencies, but they are only used for docs purpose. Since I don't have very strong opinion on that matter I will move them there.